stripe/src/main/java/com/stripe/android/paymentsheet/PaymentSheet.kt

```diff
@@ -5,20 +5,36 @@ import android.os.Parcelable
 import androidx.activity.ComponentActivity
 import androidx.fragment.app.Fragment
 import com.stripe.android.model.PaymentIntent
+import com.stripe.android.model.SetupIntent
 import com.stripe.android.paymentsheet.flowcontroller.FlowControllerFactory
 import com.stripe.android.paymentsheet.model.PaymentOption
 import kotlinx.parcelize.Parcelize
 
+/**
+ * A drop-in class that presents a bottom sheet to collect and process a customer's payment.
+ */
 class PaymentSheet internal constructor(
     private val paymentSheetLauncher: PaymentSheetLauncher
 ) {
+    /**
+     * Constructor to be used when launching the payment sheet from an Activity.
+     *
+     * @param[activity] the Activity that is presenting the payment sheet.
+     * @param[callback] called with the result of the payment after the payment sheet is dismissed.
```

Comment:
What you have is fine, but I typically see `* @ activity the Activity that is presenting the payment sheet.`

(don't bother changing if this works correctly)

Reply:
Both are valid, I can make it consistent across our codebase later. https://kotlinlang.org/docs/kotlin-doc.html#param-name

Reply:
Both are valid, I can make it consistent across our codebase later. https://kotlinlang.org/docs/kotlin-doc.html#param-name

Reply:
Done now.